glean-core/tests/string_list.rs

```diff
@@ -23,17 +26,142 @@ fn list_can_store_multiple_items() {
     let snapshot = glean.snapshot("core", false);
     assert!(snapshot.contains(r#""local.list": ["#));
     assert!(snapshot.contains(r#""first""#));
+    assert!(!snapshot.contains(r#""second""#));
```

Comment:
Yikes, we really should convert this to the proper testing functions instead of doing ugly string matching against a potential json format.

[I filed a bug so we can fix this in a follow-up](https://bugzilla.mozilla.org/show_bug.cgi?id=1557648)